src/modules/service-catalog/ServiceCatalogList.tsx

```diff
@@ -46,32 +46,13 @@ export function ServiceCatalogList({
       try {
         const response = await fetch(
           currentCursor
-            ? `/api/v2/custom_objects/service_catalog_item/records?page[size]=16&${currentCursor}`
-            : `/api/v2/custom_objects/service_catalog_item/records?page[size]=16`
+            ? `/api/v2/help_center/service_catalog/items?page[size]=16&${currentCursor}`
+            : `/api/v2/help_center/service_catalog/items?page[size]=16`
```

Comment:
I notice that the page[size] parameter is hardcoded to `16` is that intended?

Reply:
I changed it to some const variable so we can change it easily.